.github/workflows/phylogenetic-ncbi.yaml

```diff
@@ -95,10 +117,11 @@ jobs:
           --memory 28800mib \
           . \
             deploy_all \
-            --configfile config/h5n1-cattle-outbreak.yaml \
+            --configfile config/${CONFIG_NAME}.yaml \
             --config "${config[@]}"
 
       env: |
         NEXTSTRAIN_DOCKER_IMAGE: ${{ inputs.image }}
         TRIAL_NAME: ${{ inputs.trial-name }}
+        CONFIG_NAME: ${{ inputs.config-yaml }}
       artifact-name: phylogenetic-full-genome-build-output
```

Comment:
I'm not sure what these artifacts are used for, and since a run of the action is either D1.1 or cattle-outbreak I think there'll be no overwriting of the hardcoded name, but wanted to point this out.

Reply:
I've used the artifacts to debugging the workflows that are **only** run in the docker runtime via GH Actions. Not as relevant here since the job is running in AWS Batch and we have logs and workflow intermediates saved on S3. 

I _think_ the D1.1 and cattle-outbreak artifacts will overwrite each other within a run of `ingest-to-phylogenetic-ncbi.yaml`, so we could interpolate the `inputs.config-yaml` for them to be unique. 

```
artifact-name: phylogenetic-${{ inputs.config-yaml }}-outputs
```

Reply:
> I think the D1.1 and cattle-outbreak artifacts will overwrite each other within a run 

The way it's set up is that you _can't_ run both in a single action instance. But this is a feature I'd like to enable in the future when both cattle-flu and D1.1 are running from the same config YAML and therefore possible to run  within a single `snakemake` graph. Ok to revisit this then?

Reply:
Ah, I did not mean at the Snakemake workflow level, I'm talking about the GH Action workflow. Since they are both called within `ingest-to-phylogenetic-ncbi.yaml`, that means they are running in one GH Action workflow so the artifact zip archives will overwrite each other.

Reply:
>  Since they are both called within ingest-to-phylogenetic-ncbi.yaml, that means they are running in one GH Action workflow so the artifact zip archives will overwrite each other.

Ah! Of course. I was thinking in the context of the `phylogenetic-ncbi.yaml` level where they are independent runs. Will change now.

Reply:
[Changed](https://github.com/nextstrain/avian-flu/compare/5843d0cab6fa752e066a02bc996c717fab315b98..58d2c4bc047f11655616ae7286c8f6d78757b5ce) to `artifact-name: phylogenetic-${{ inputs.config-yaml }}-outputs`